gemrb/core/Game.cpp

```diff
@@ -40,8 +40,27 @@
 #include "GUI/GameControl.h"
 #include "System/DataStream.h"
 #include "System/StringBuffer.h"
+#include <iterator>
+#include <vector>
+#include <algorithm>
 
 namespace GemRB {
+	
+	
+struct HealingResource {
```

Comment:
IfI put them inside CastOnRest() I just get a bunch of errors. Possibly related that I put them in a container.